bsdlib/include/bsd_limits.h

```diff
@@ -10,6 +10,14 @@
  * @ingroup bsd_library
  * @{
  * @brief Upper and lower bound limits of BSD library.
+ *
+ * @details This module provides:
+ * -# A common entry point for limits specific system initializations.
+ * -# Limits specific parameters:
+ *         -# Maximum number of concurrent sockets in the system.
+ *         -# Maximum number of concurrent secure sockets in the system.
+ *         -# Maximum number AT sockets and subscriptions for AT events in the system.
+ *         -# ..
```

Comment:
I don't think we should leave it like this.

Reply:
I am not sure why this ended up in the diff, when I look back in the source repo this file has always been like this. It has just been fixed, but I can just remove it so it looks like how the diff say the previous version looked.